packages/docs-app/src/examples/core-examples/tagInputExample.tsx

```diff
@@ -92,11 +92,12 @@ export class TagInputExample extends React.PureComponent<IExampleProps, ITagInpu
     private renderOptions() {
         return (
             <>
+                <H5>Props</H5>
                 <Switch label="Fill container width" checked={this.state.fill} onChange={this.handleFillChange} />
                 <Switch label="Large" checked={this.state.large} onChange={this.handleLargeChange} />
                 <Switch label="Disabled" checked={this.state.disabled} onChange={this.handleDisabledChange} />
                 <Switch label="Add on blur" checked={this.state.addOnBlur} onChange={this.handleAddOnBlurChange} />
-                <H5>Tag props</H5>
+                <H5>Example</H5>
```

Comment:
this should stay as "Tag props" -- these switches affect `tagProps` behavior, which is one of the props